pkg/api/accesscontrol.go

```diff
@@ -419,8 +408,8 @@ func (hs *HTTPServer) declareFixedRoles() error {
 	}
 
 	return hs.AccessControl.DeclareFixedRoles(
-		provisioningWriterRole, datasourcesReaderRole, datasourcesWriterRole, datasourcesIdReaderRole,
-		datasourcesCompatibilityReaderRole, orgReaderRole, orgWriterRole,
+		provisioningWriterRole, datasourcesReaderRole, datasourcesWriterRole,
+		datasourcesIdReaderRole, orgReaderRole, orgWriterRole,
```

Comment:
Do we need a migration to remove the role and its grants from the database? 
Seeding should take care of removing the permissions from the basic roles natively.

Reply:
Yes i seems like we have to remove that role, We could do it in another pr